tests/routes/post_id_check_application_update_test.py

```diff
@@ -0,0 +1,52 @@
+from tests.conftest import TestClient
+from unittest.mock import patch
+
+class Post:
+    class Returns200:
+        @patch('routes.beneficiaries.beneficiary_job.delay')
+        def when_has_exact_payload(self, mocked_beneficiary_job, app):
+            # Given
+            data = {'id': '5'}
+
+            # When
+            response = TestClient(app.test_client()) \
+                .post('/beneficiaries/application_update', json=data)
+
+            # Then
+            assert response.status_code == 200
+            mocked_beneficiary_job.assert_called_once_with(5)
+
+    class Returns400:
+        @patch('routes.beneficiaries.beneficiary_job.delay')
+        def when_no_payload(self, mocked_beneficiary_job, app):
+            # When
+            response = TestClient(app.test_client()) \
+                .post('/beneficiaries/application_update')
+
+            # Then
+            assert response.status_code == 400
```

Comment:
Petite suggestion, pourquoi pas aussi (sur tous ceux de la class Returns400):
```suggestion
            assert response.status_code == 400
            mocked_beneficiary_job.assert_not_called()
```